src/index.vue

```diff
@@ -8,6 +8,10 @@ export default {
     event: {
       type: String,
       default: "keyup"
+    },
+    preventDefault: {
+      type: Bolean,
+      default: null
```

Comment:
This line can & should be deleted.

Boolean types are by default `false` if not defined, so we can omit this line completely.